src/dut.rs

```diff
@@ -143,6 +143,7 @@ lazy_static! {
     static ref DUT_ATTRIBUTE_CMDS: HashMap<&'static str, &'static str> = {
         let mut m: HashMap<&'static str, &'static str> = HashMap::new();
         m.insert("board", r"cat /etc/lsb-release | grep CHROMEOS_RELEASE_BOARD | cut -d '=' -f 2 | cut -d '-' -f 1");
+        // VM instances do not have a valid HWID.
```

Comment:
Could you add 'NOTE: '  at the beginning of this comment?

Reply:
Fixed!